project/Build.scala

```diff
@@ -1335,7 +1335,8 @@ object Build {
       BuildInfoPlugin.buildInfoDefaultSettings
 
   lazy val presentationCompilerSettings = {
-    val mtagsVersion = "1.3.0+56-a06a024d-SNAPSHOT"
+    // Avoid bumping to snapshot since it's in a different repository than maven central
+    val mtagsVersion = "1.3.1"
 
     Seq(
       resolvers ++= Resolver.sonatypeOssRepos("snapshots"),
```

Comment:
According to comment, are we going to stop using nightly here ? If so lets remove this line.